_i18n/en.yml

```diff
@@ -351,8 +351,8 @@ help_pages:
         {: .number-list}
 
       ## Related articles
-      * [Authentication options](#)
-      * [Verify your identity](#)
+      * [Authentication options](site.baseurl/help/get-started/authentication-options/)
+      * [Verify your identity](site.baseurl/help/verify-your-identity/how-to-verify-your-identity/)
```

Comment:
not strictly related to this request but I figured I'd update it while I was here